src/CiteProc/Render.php

```diff
@@ -83,45 +83,56 @@ public function bibliography(string $locale, array $styleNames, array $jsonData)
       $style_iterator = 0;
       foreach ($styleNames as $selected_style) {
         $style = StyleSheet::loadStyleSheet($selected_style);
+        $style_xml = simplexml_load_string($style);
```

Comment:
@aksm ups. Do you still need these lines?

Reply:
I don't see this is used, might be left over of your previous code. Can you double check and if not remove? Thanks!

Reply:
Oops, fixed!